docker/init/03-users.js

```diff
@@ -1,75 +1,25 @@
-const users = db.getSiblingDB('trello-clone')
-
 const user1 = {
   _id: ObjectId('6230ef5c91ecd051dc75df37'),
-  username: 'Marek Kowalski',
+  username: 'Marek Kowalski2137',
   passwordHash: 'pass123hash1',
   avatarId: 23,
-  dashboards: [
```

Comment:
what?! why? :o

Reply:
I think I didn't quite get you when you said `change memberOfDashboards and use ID` https://github.com/kacper-cyra/camp-trello-clone/pull/112#discussion_r862524643 😬